scripts/release/publish-docs-artifacts.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Publish material2 docs assets to the material2-docs-content repo
+# material.angular.io will pull from this assets repo to get the latest docs
+
+cd "$(dirname $0)/../../"
+
+docsPath="./dist/docs"
+tempRepoPath="./dist/tempRepo"
+repoUrl="https://github.com/angular/material2-docs-content"
```

Comment:
Okay so here is the repository: `https://github.com/DevVersion/material2-docs-content.git`

@jelbourn In the future we can probably switch to another token.